meilisearch-http/src/analytics/segment_analytics.rs

```diff
@@ -269,7 +290,7 @@ impl Segment {
                         },
                     })),
                     user: self.user.clone(),
-                    traits: Self::compute_traits(&self.opt, stats),
+                    traits: dbg!(Self::compute_traits(&self.opt, stats)),
```

Comment:
Can you remove this `dbg!` too?
```suggestion
                    traits: Self::compute_traits(&self.opt, stats),
```

Reply:
Yes sorry I left this PR a bit unfinished while waiting for the cli options to be merged :grin: